DynamicAnalyzer/views/android/environment.py

```diff
@@ -40,41 +40,49 @@ def wait(self, sec):
         logger.info('Waiting for %s seconds...', str(sec))
         time.sleep(sec)
 
+    def proof_connect(self, output):
+        """Proof if connection output is not containing errors"""
+        if b'unable to connect' in output or b'failed to connect' in output:
+            logger.error('%s', output.decode('utf-8').replace('\n', ''))
+            return False
+        else:
+            return True
+
+    def run_subprocess_verify_output(self, command):
+        out = subprocess.check_output(command)
+        self.wait(2)
+        if not proof_connect(out):
```

Comment:
neat way is to `return self.proof_connect(out)`

Reply:
good idea, this makes it even smaller